app/services/github/repositories/sync.rb

```diff
@@ -13,15 +13,10 @@ def call
         Rails.logger.info("[GH] -- Repositories: #{repositories.count}")
 
         repositories.map do |repository|
-          repository_owner, repository_name = repository.link.split('/')[-2..-1]
+          update_repository_languages(repository)
+          update_repository_stats(repository)
```

Comment:
since we have two updates here, I think we can use `transaction` block. If everything should get updated if the request succeed right?

Reply:
I think that if one of them does not update the some of the infos: languages or stars and issues, this is not a problem, cause these infos are optional.